master/security/service-accounts.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Use service accounts in policy rules
+---
+
+### Big picture
+
+Use Calico network policy to allow/deny traffic for Kubernetes services. 
+
+### Value
+
+Using Calico network policy, you can leverage Kubernetes service accounts with RBAC for flexible control over how policies are applied in a cluster. For example, the network security can have RBAC permissions to:
+
+- Control which service accounts the developer team can use within a namespace
+- Write high-priority network policies for those service accounts (that the developer team cannot override) 
```

Comment:
> - Control which service accounts the developer team can use within a namespace
> - Write high-priority network policies for those service accounts (that the developer team cannot override) 

Maybe impeded by the sentence above, but I don't know if this is right. Nothing about service account policies prevents developers from creating lower order (higher priority) network policies.

Reply:
This is Alex text.  It is just an example of how customers could use RBAC to control how policies are applied to a cluster. Not sure how to make this more clear...ideas?